api/apiclient.go

```diff
@@ -617,7 +617,9 @@ func gorillaDialWebsocket(ctx context.Context, urlStr string, tlsConfig *tls.Con
 	// TODO(rogpeppe) We'd like to set Deadline here
 	// but that would break lots of tests that rely on
 	// setting a zero timeout.
-	netDialer := net.Dialer{}
+	netDialer := net.Dialer{
+		KeepAlive: 30 * time.Second,
```

Comment:
Can we do this as a const at the top of the file where all the other such ones live.
Also, we have pingTimeout defined as 30s also. Maybe use the same value for keep alive?